2020/questions.md

```diff
@@ -331,22 +337,19 @@ NEW question
 
 ## Part 5
 ### Part 5a: Value of Companies and Foundations
-32. To what degree do you perceive the following companies to be “good open source community citizens” in terms of their contributions, collaboration and leadership on open source projects and initiatives within the open source ecosystem? 
+32. "The following list represents a range of large companies that participate in open source communities. To what degree do you perceive each of them to be “good open source community citizens” in terms of contributions, collaboration and leadership on open source projects and initiatives within the open source ecosystem?"
+To what degree do you perceive the following companies to be “good open source community citizens” in terms of their contributions, collaboration and leadership on open source projects and initiatives within the open source ecosystem? 
 >{Excellent Above Average Average Below Average Very Poor Don't know}
 * AWS
+* Comcast 
+* Facebook
 * Google
-* Microsoft
```

Comment:
Microsoft should stay in list. Logic for removing?

Reply:
Microsoft was kept in -- for some reason, I don't know why it looks this way in the GitHub before vs after comparsion.